git-cliff-core/src/changelog.rs

```diff
@@ -62,9 +62,9 @@ impl<'a> Changelog<'a> {
 			config,
 			additional_context: HashMap::new(),
 		};
+		changelog.add_remote_data()?;
```

Comment:
This was added in #703 - I'm not sure if changing the order would affect things but I'm guessing it should be fine.

Reply:
I looked at [add_remote_data](https://github.com/orhun/git-cliff/blob/32da6d390394c0907317b0da0b8f4d2c72ec2fa7/git-cliff-core/src/changelog.rs#L420-L481) and found no relevant field read access that would have been populated by `process_commits` or `process_releases`.